internal/runtime/runtime.go

```diff
@@ -115,26 +119,58 @@ func (gr *GolangRuntime) ProcessMessage(edgexcontext *appcontext.Context, envelo
 	// dereference to pointer to the object
 	target = reflect.ValueOf(target).Elem().Interface()
 
-	var result interface{}
-	var continuePipeline = true
-
 	// Make copy of transform functions to avoid disruption of pipeline when updating the pipeline from registry
 	gr.isBusyCopying.Lock()
 	transforms := make([]appcontext.AppFunction, len(gr.transforms))
 	copy(transforms, gr.transforms)
 	gr.isBusyCopying.Unlock()
 
-	for index, trxFunc := range transforms {
-		if result != nil {
-			continuePipeline, result = trxFunc(edgexcontext, result)
-		} else {
+	return gr.executePipeline(target, contentType, edgexcontext, transforms, 0, false)
+
+}
+
+// Initialize sets the internal reference to the StoreClient for use when Store and Forward is enabled
+func (gr *GolangRuntime) Initialize(storeClient interfaces.StoreClient) {
```

Comment:
Initialize() is only ever called from production code with storeClient == nil.  Is this intentional?

Reply:
Yes, just so that I could split the PRs in to smaller chunks. One of the next PRs with change that.